pkg/cloud/services/autoscaling/autoscalinggroup.go

```diff
@@ -238,10 +241,12 @@ func (s *Service) DeleteASGAndWait(name string) error {
 	s.scope.Debug("Waiting for ASG to be deleted", "name", name)
 
 	input := &autoscaling.DescribeAutoScalingGroupsInput{
-		AutoScalingGroupNames: aws.StringSlice([]string{name}),
+		AutoScalingGroupNames: []string{name},
 	}
 
-	if err := s.ASGClient.WaitUntilGroupNotExistsWithContext(context.TODO(), input); err != nil {
+	waiter := autoscaling.NewGroupNotExistsWaiter(s.ASGClient)
+
+	if err := waiter.Wait(context.TODO(), input, 15*time.Minute); err != nil {
```

Comment:
Where did the 15 minute value come from?

Reply:
It's just a value that I thought might be reasonable